### PR TITLE
admin dashboard

### PR DESCRIPTION
**Pull request by yscope75**

### CODE REVIEW

**Comment by yscope75** [comment]
screenshot from 2016-02-22 16-20-53

File: app/controllers/admin/dashboard_controller.rb [hunk 1]
```diff
@@ -1,7 +1,13 @@
...
+  before_action :set_student, :set_partner, :set_student_hero
 
+  def set_student 
+    @students = Student.all
+  end
+  def set_partner
+    @partners = Partner.all
+  end
+  def set_student_hero
+    @heros = Student.where(student_type: "hero")
```
Comment (lhhieu89) [inline app/controllers/admin/dashboard_controller.rb]: Please use `enum` convention
```ruby
@heroes = Student.hero
```

**Comment by lhhieu89** [comment]
@yscope75 : Please remove file `app/models/user.rb.orig`

File: app/controllers/admin/dashboard_controller.rb [hunk 2]
```diff
@@ -1,7 +1,13 @@
...
+  before_action :set_student, :set_partner, :set_student_hero
 
+  def set_student 
+    @students = Student.all
+  end
+  def set_partner
+    @partners = Partner.all
+  end
+  def set_student_hero
+    @heros = Student.hero
```
Comment (lhhieu89) [inline app/controllers/admin/dashboard_controller.rb]: i commented that use `@heroes` before, right?

**Pull request merged**